backend/tracim_backend/command/cleanup.py

```diff
@@ -1,23 +1,51 @@
 import argparse
 from pyramid.scripting import AppEnvironment
+from sqlalchemy.engine import Engine
+from sqlalchemy.event import listen
 import traceback
 import typing
 
-from tracim_backend import UserDoesNotExist
 from tracim_backend.applications.agenda.models import AgendaResourceType
 from tracim_backend.apps import AGENDA__APP_SLUG
 from tracim_backend.command import AppContextCommand
 from tracim_backend.exceptions import AgendaNotFoundError
+from tracim_backend.exceptions import CannotDeleteUniqueRevisionWithoutDeletingContent
+from tracim_backend.exceptions import ContentNotFound
+from tracim_backend.exceptions import ContentRevisionNotFound
 from tracim_backend.exceptions import UserCannotBeDeleted
+from tracim_backend.exceptions import UserDoesNotExist
+from tracim_backend.exceptions import WorkspaceNotFound
 from tracim_backend.extensions import app_list
 from tracim_backend.lib.cleanup.cleanup import CleanupLib
 from tracim_backend.lib.cleanup.cleanup import UserNeedAnonymization
 from tracim_backend.lib.core.application import ApplicationApi
+from tracim_backend.lib.core.content import ContentApi
 from tracim_backend.lib.core.user import UserApi
+from tracim_backend.lib.core.workspace import WorkspaceApi
 from tracim_backend.models.auth import User
 from tracim_backend.models.tracim_session import unprotected_content_revision
 
 
+# INFO - F.S - 2023-05-30 - Enable cascade delete on sqlite database
+def set_sqlite_pragma_on(dbapi_connection, connection_record):
+    cursor = dbapi_connection.cursor()
+    try:
+        cursor.execute("PRAGMA foreign_keys=ON")
+    except Exception:
+        pass
+    cursor.close()
+
+
+# INFO - F.S - 2023-06-05 - Disable cascade delete on sqlite database
+def set_sqlite_pragma_off(dbapi_connection, connection_record):
```

Comment:
@floreal15 Can you rename these functions (`set_sqlite_pragma_on` and `set_sqlite_pragma_off`) ? The job is to disable/enable foreign keys, so I suggest renaming into something like `disable_sqlite_foreign_keys` and ``enable_sqlite_foreign_keys``

Reply:
yes, i have done it